src/main/kotlin/minesweeper/domain/MineSweeperBoardFactory.kt

```diff
@@ -0,0 +1,35 @@
+package minesweeper.domain
+
+fun factory(block: MineSweeperBoardBuilder.() -> Unit): MineSweeperBoard {
+    return MineSweeperBoardBuilder().apply(block).build()
+}
+
+class MineSweeperBoardBuilder {
+    private lateinit var height: Height
```

Comment:
```suggestion
    private var height: Height by Delegates.notNull()
```

value class가 안되는 이유는 primitive에 대해서 lateinit을 지원하고 있지 않습니다 😊
primitive type에 대해서는 lateinit에 대해 필요하면 Delegates.notNull() 을 사용하도록 하고 있어요.

참고
- [notNull delegate vs lateinit](https://discuss.kotlinlang.org/t/notnull-delegate-vs-lateinit/1923)

Reply:
아하!... 이런 이유 때문이었군요?  자꾸 습관적으로 value class를 래퍼 클래스라고 생각하게 되네요. 👀